frontend/src/components/pages/overview/Overview.tsx

```diff
@@ -261,8 +263,10 @@ const ResourcesBadge = (p: { type?: string | undefined }) => {
 function ClusterDetails() {
     const overview = api.clusterOverview;
     const brokers = api.brokers;
-    if (!overview || !brokers)
-        return <Skeleton paragraph={{ rows: 16 }} />
+
+    if (!overview || !brokers) {
+        return <SkeletonText mt={5} noOfLines={13} spacing={5} skeletonHeight={4} speed={0} />
```

Comment:
16 -> 13?
also is it intentional that here we have speed = 0

Reply:
For `Skeleton` you need to provide `isActive` or sth like that, for `SketetonText` it's the other way around, you need to disable animation with speed=0, so I think this is correct.